### PR TITLE
Update EIP-5023: Move to Review, minor corrections to interface

### PR DESCRIPTION
**Pull request by yaruno**
Update to interface hash id comment.
Update to function syntax on interface.
Updated the Draft status to Review.

### CODE REVIEW

File: EIPS/eip-5023.md [hunk 1]
```diff
@@ -37,14 +37,14 @@ We believe that the wider audience should benefit from an abstraction level high
...
 
   /// @dev This emits when a token is shared, reminted and given to another wallet that isn't function caller
   event Share(address indexed from, address indexed to, uint256 indexed tokenId, uint256 derivedFromtokenId);
 
   /// @dev Shares, remints an existing token, gives a newly minted token a fresh token id, keeps original token at function callers possession and transfers newly minted token to receiver which should be another address than function caller. 
-  function share(address to, uint256 tokenIdToBeShared) returns(uint256 newTokenId) external;
+  function share(address to, uint256 tokenIdToBeShared) external returns(uint256 newTokenId);
 
 } 
 ```
```
Comment (Pandapip1) [inline EIPS/eip-5023.md:50]: Line 52: there is a missing "The".

File: EIPS/eip-5023.md [hunk 2]
```diff
@@ -66,6 +66,7 @@ B -> A -> C
...
 
 These tree structures can be further aggregated and collapsed to network representations e.g. social graphs on basis of whom has shared to whom over a span of time. E.g. if Bob shared a token to Alice, and Alice has shared a different token to Charlie and Bob has shared a token to Charlie, connections form between all these parties through sharing activities.
+
 ```
```
Comment (Pandapip1) [inline EIPS/eip-5023.md:70]: Nit: specify language
```suggestion
```text
```

File: EIPS/eip-5023.md [hunk 3]
```diff
@@ -57,7 +57,7 @@ Current NFT standards define transferable non-fungible tokens, but not shareable
...
 
-Over time, a tree-like structures can be formed from the Share event information. If Bob shared to Alice, and Alice shared further to Charlie and Alice also shared to David a rudimentary tree structure forms out from sharing activity. This share event data can be later on utilized to gain more information of share activities that the tokens represent. 
+Over time, a tree-like structures can be formed from the Share event information. If Bob shared to Alice, and Alice shared further to Charlie and Alice also shared to David a rudimentary tree structure forms out from sharing activity. This share event data can be later on utilized to gain more information of share activities that the tokens represent.
 
 ```
```
Comment (Pandapip1) [inline EIPS/eip-5023.md:62]:
```suggestion
```text
```
Nit: specify language

File: EIPS/eip-5023.md [hunk 4]
```diff
@@ -66,6 +66,7 @@ B -> A -> C
...
 
 These tree structures can be further aggregated and collapsed to network representations e.g. social graphs on basis of whom has shared to whom over a span of time. E.g. if Bob shared a token to Alice, and Alice has shared a different token to Charlie and Bob has shared a token to Charlie, connections form between all these parties through sharing activities.
+
 ```
  B----A----C         
   \_______/
```
Comment (Pandapip1) [inline EIPS/eip-5023.md:72]: Nit on L83: missing solidity language

File: EIPS/eip-5023.md [hunk 5]
```diff
@@ -159,6 +160,7 @@ contract ShareableERC721 is ERC721URIStorage, Ownable, IERC5023 /* EIP165 */ {
```
Comment (Pandapip1) [inline EIPS/eip-5023.md:161]:
```suggestion
```
Not sure why you need so many blank lines.

File: EIPS/eip-5023.md [hunk 6]
```diff
@@ -159,6 +160,7 @@ contract ShareableERC721 is ERC721URIStorage, Ownable, IERC5023 /* EIP165 */ {
 
 
 ```
+
 ## Security Considerations
 
 TBD
```
Comment (Pandapip1) [inline EIPS/eip-5023.md:166]:
```suggestion
Needs discussion.
```

**Review by Pandapip1 — CHANGES_REQUESTED** [review]

File: EIPS/eip-5023.md (hunk 1 above)
Comment (yaruno) [reply EIPS/eip-5023.md:50]: Added The to Line52.

**Review by Pandapip1 — APPROVED** [review]

**Pull request merged**